starboard/tools/testing/test_runner.py

```diff
@@ -998,7 +980,9 @@ def Abort(signum, frame):
   if args.run:
     run_success = runner.RunAllTests()
 
-  runner.GenerateCoverageReport()
+  if args.coverage_report and coverage_directory:
```

Comment:
Unless I'm mistaken, this looks like it will run cov report even if it didn't build or run successfully. Do we want coverage in that case or should we gate it to successful runs?

Reply:
Agreed, if the tests failed the coverage info might not be representative.